src/app/Instance/BridgeOverview/BridgeOverview.test.tsx

```diff
@@ -77,15 +77,14 @@ describe("Bridge Overview", () => {
     await waitForI18n(comp);
 
     expect(comp.queryByText("No processors")).not.toBeInTheDocument();
-    expect(comp.queryByText("Processor one")).toBeInTheDocument();
-    expect(comp.queryByText("Processor two")).toBeInTheDocument();
-    expect(comp.queryByText("Processor three")).toBeInTheDocument();
-    expect(comp.queryByText("Processor four")).toBeInTheDocument();
-
-    expect(comp.getByRole("link", { name: "Processor one" })).toHaveAttribute(
-      "href",
-      "/instance/3543edaa-1851-4ad7-96be-ebde7d20d717/processor/a72fb8e7-162b-4ae8-9672-f9f5b86fb3d7"
-    );
+
+    processorData.map((processor) => {
+      expect(comp.queryByText(processor.name)).toBeInTheDocument();
+      expect(comp.getByRole("link", { name: processor.name })).toHaveAttribute(
+        "href",
+        processor.url
+      );
```

Comment:
Following my previous comment, I would prefer to check against a url you compose directly here, instead of retrieving it from the processor list. Just to make things more clear.

More or less like this:
 
```suggestion
      expect(comp.getByRole("link", { name: processor.name })).toHaveAttribute(
        "href",
        `/instance/${instanceId}/processor/${processor.id}"`
      );
```

Wdyt?

Reply:
Yes. This gives more clarity. Thank you for the suggestion. I will update it.